libbess-python/bess.py

```diff
@@ -10,7 +10,7 @@
 import time
 
 import service_pb2
-import proto_conv
+from protobuf_to_dict import *
```

Comment:
It is generally recommended not to use wildcard import. How about `from protobuf_to_dict import protobuf_to_dict, dict_to_protobuf`?